product-backend/src/main/java/org/kainos/ea/db/AdminDao.java

```diff
@@ -0,0 +1,33 @@
+package org.kainos.ea.db;
+
+import org.kainos.ea.cli.Admin;
+
+import java.sql.*;
+
+public class AdminDao {
+    private DatabaseConnector databaseConnector = new DatabaseConnector();
+
+    public int createBand(Admin admin) throws SQLException {
+        Connection c = databaseConnector.getConnection();
+
+
+        String insertStatement = "INSERT INTO Band (name, level, responsibilities) VALUES (?, ?, ?)";
+
+        PreparedStatement st = c.prepareStatement(insertStatement, Statement.RETURN_GENERATED_KEYS);
+
+        st.setString(1, admin.getName());
+        st.setString(2, admin.getLevel());
+        st.setString(3, admin.getResponsibilities());
+
+
+        st.executeUpdate();
+
+        ResultSet rs = st.getGeneratedKeys();
+
+        if (rs.next()) {
+            return rs.getInt(1);
+        }
+        return -1;
+    }
```

Comment:
If you want to do it in a bit more elegant way you can use Optionals
For primitive types there are separate classes e.g. OptionalInt
https://docs.oracle.com/javase/8/docs/api/java/util/OptionalInt.html


```suggestion
    public OptionalInt createBand(Admin admin) throws SQLException {
        Connection c = databaseConnector.getConnection();


        String insertStatement = "INSERT INTO Band (name, level, responsibilities) VALUES (?, ?, ?)";

        PreparedStatement st = c.prepareStatement(insertStatement, Statement.RETURN_GENERATED_KEYS);

        st.setString(1, admin.getName());
        st.setString(2, admin.getLevel());
        st.setString(3, admin.getResponsibilities());


        st.executeUpdate();

        ResultSet rs = st.getGeneratedKeys();

        if (rs.next()) {
            return OptionalInt.of(rs.getInt(1));
        }
        return Optional.empty();
    }
```

Then in the AdminService you do
```
try {
            OptionalInt id = adminDao.createBand(admin);

            return id.orElseThrow();
        } catch (SQLException e) {
            System.err.println(e.getMessage());
            throw new FailedToCreateBandException();
        } catch (NoSuchElementException e) {
            throw new FailedToCreateBandException(e);
        }
```